nvi-rest/src/test/java/no/sikt/nva/nvi/rest/BaseCandidateRestHandlerTest.java

```diff
@@ -0,0 +1,298 @@
+    Candidate.upsert(request, candidateRepository, periodRepository);
+    return Candidate.fetchByPublicationId(
+        request::publicationId, candidateRepository, periodRepository);
+  }
+
+  protected static VerifiedNviCreatorDto setupVerifiedCreator(
+      URI id, Collection<URI> affiliations, URI topLevelInstitutionId) {
+    affiliations.forEach(
+        affiliation ->
+            mockOrganizationResponseForAffiliation(
+                topLevelInstitutionId, affiliation, mockUriRetriever));
+    return new VerifiedNviCreatorDto(id, List.copyOf(affiliations));
+  }
+
+  private static InstitutionPoints getCreatorPoints(
+      Map.Entry<URI, Collection<NviCreatorDto>> entry) {
+    var institution = entry.getKey();
+    var creators = entry.getValue();
+    var creatorPoints = getCreatorPoints(creators);
+    var pointsPerInstitution =
+        creatorPoints.stream().map(CreatorAffiliationPoints::points).reduce(ZERO, BigDecimal::add);
+    return new InstitutionPoints(institution, pointsPerInstitution, creatorPoints);
+  }
+
+  private static List<CreatorAffiliationPoints> getCreatorPoints(
+      Collection<NviCreatorDto> creators) {
+    return creators.stream()
+        .filter(VerifiedNviCreatorDto.class::isInstance)
+        .map(VerifiedNviCreatorDto.class::cast)
+        .map(BaseCandidateRestHandlerTest::getCreatorPoints)
+        .toList();
+  }
+
+  private static CreatorAffiliationPoints getCreatorPoints(VerifiedNviCreatorDto creator) {
+    return new CreatorAffiliationPoints(
+        creator.id(), creator.affiliations().getFirst(), randomBigDecimal());
+  }
+
+  protected Candidate setupNonApplicableCandidate(URI institutionId) {
+    var candidate = upsert(createUpsertCandidateRequest(institutionId).build());
+    return Candidate.updateNonCandidate(
+            createUpsertNonCandidateRequest(candidate.getPublicationId()), candidateRepository)
+        .orElseThrow();
+  }
+
+  protected Candidate setupCandidateWithUnverifiedCreator(URI institutionId) {
+    var verifiedCreator = setupDefaultVerifiedCreator();
+    var unverifiedCreator = setupDefaultUnverifiedCreator();
+    var request =
+        createUpsertCandidateRequestWithPoints(
+                Map.of(institutionId, List.of(verifiedCreator, unverifiedCreator)))
+            .build();
+    return upsert(request);
+  }
+
+  protected UnverifiedNviCreatorDto setupDefaultUnverifiedCreator() {
+    return setupUnverifiedCreator(
+        randomString(), List.of(subUnitCristinOrgId), topLevelCristinOrgId);
+  }
+
+  protected static UnverifiedNviCreatorDto setupUnverifiedCreator(
+      String name, Collection<URI> affiliations, URI topLevelInstitutionId) {
+    affiliations.forEach(
+        affiliation ->
+            mockOrganizationResponseForAffiliation(
+                topLevelInstitutionId, affiliation, mockUriRetriever));
+    return new UnverifiedNviCreatorDto(name, List.copyOf(affiliations));
+  }
+
+  // Not used (yet)
```

Comment:
In general, mark these with `// TODO: ` as these are picked up as warnings in the IDE, which is a bit more of a reminder to do what we intend, even if we come back to the task after a month of fever dreams

Reply:
Good point! These are being removed in another PR today, but I'll keep it in mind.